packages/amplication-client/src/Project/ProjectListItem.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import InnerTabLink from "../Layout/InnerTabLink";
+import { Project } from "../models";
+
+type Props = {
+  project: Project;
+  onProjectSelected: (project: Project) => void;
```

Comment:
this is not being used

Reply:
Fixed. I don't know why it is not marked as outdated